osu.Game/Screens/OnlinePlay/Multiplayer/Participants/ParticipantsList.cs

```diff
@@ -55,6 +59,19 @@ protected override void OnRoomUpdated()
                 // Add panels for all users new to the room.
                 foreach (var user in Room.Users.Except(panels.Select(p => p.User)))
                     panels.Add(new ParticipantPanel(user));
+
+                if (currentHostPanel == null || !currentHostPanel.User.Equals(Room.Host))
+                {
+                    // Reset position of previous host back to normal, if one existing.
+                    if (currentHostPanel != null && panels.Contains(currentHostPanel))
+                        panels.SetLayoutPosition(currentHostPanel, 0);
+
+                    currentHostPanel = null;
+
+                    // Change position of new host to display above all participants.
+                    if (Room.Host != null)
+                        panels.SetLayoutPosition(currentHostPanel = panels.SingleOrDefault(u => u.User.Equals(Room.Host)), -1);
```

Comment:
Why is this `SingleOrDefault()` now? If the "default" logic ever kicks in, then this is now calling `SetLayoutPosition` on a null drawable, no?

Reply:
*starting to lose my mind on which allows null and which doesn't*.

Thought we might as well just use `SingleOrDefault` in case the host isn't part of the panels for one reason or another (since all multiplayer hub events go through one central `OnRoomUpdated` method), but didn't take the nullability in that method into account (we might want to speed up the process of switching to NRT :p).

Reply:
Well sure, you can set `currentHostPanel` to null. Just don't set `SetLayoutPosition()` on that null.